packages/beacon-node/test/spec/presets/fork_choice.test.ts

```diff
@@ -193,7 +215,33 @@ const forkChoiceTest =
 
               try {
                 let blockImport;
-                if (config.getForkSeq(slot) >= ForkSeq.deneb) {
+                const forkSeq = config.getForkSeq(slot);
+
+                if (forkSeq >= ForkSeq.fulu) {
+                  if (columns === undefined) {
+                    columns = [];
+                  }
+
+                  for (const column of columns) {
+                    verifyDataColumnSidecar(column);
+                    verifyDataColumnSidecarInclusionProof(column);
+                    await verifyDataColumnSidecarKzgProofs(
+                      column.kzgCommitments,
+                      Array.from({length: column.column.length}, () => column.index),
+                      column.column,
+                      column.kzgProofs
+                    );
+                  }
+
+                  const blockData = {
+                    fork,
+                    dataColumns: columns,
+                    dataColumnsBytes: columns.map(() => null),
+                    dataColumnsSource: DataColumnsSource.gossip,
+                  } as BlockInputDataColumns;
+
+                  blockImport = getBlockInput.availableData(config, signedBlock, BlockSource.gossip, blockData);
+                } else if (forkSeq >= ForkSeq.deneb && forkSeq < ForkSeq.fulu) {
```

Comment:
`} else if (forkSeq >= ForkSeq.deneb) {` would be sufficient as if fork is post fulu we execute if branch